README.md

```diff
@@ -66,15 +66,29 @@ Initializing the SDK should be done in native code inside your `AppDelegate` (iO
 @import AEPServices;
 @import AEPSignal;
 @import AEPLifecycle;
-@import AEPIdentity;
-@import AEPUserProfile;
+@import AEPEdge;
```

Comment:
How about add // AppDelegate.h before the import statement, so it is clearer  for use  to know where to add  it.